pkg/bindings/system/system.go

```diff
@@ -41,7 +42,7 @@ func Events(ctx context.Context, eventChan chan entities.Event, cancelChan chan
 		}()
 	}
 
-	dec := json.NewDecoder(response.Body)
+	dec := jsoniter.NewDecoder(response.Body)
```

Comment:
I think this is missing a `var json = jsoniter.ConfigCompatibleWithStandardLibrary`
Not sure if there will be a noticeable difference between the two.

Reply:
Because `errorhandling` pkg is imported this should get the correct decoders.